airbyte-integrations/connectors/source-quickbooks/source_quickbooks/__init__.py

```diff
@@ -3,7 +3,6 @@
 #
 
 
-from .components import CustomDatetimeBasedCursor
 from .source import SourceQuickbooks
 
-__all__ = ["SourceQuickbooks", "CustomDatetimeBasedCursor"]
+__all__ = ["SourceQuickbooks"]
```

Comment:
I think you can just remove the line altogether, and make this file empty. The file is still required, but I think no content is needed.

Reply:
Sure, I'll do that